src/main/java/com/thebois/Pawntastic.java

```diff
@@ -100,9 +109,23 @@ private void setUpUserInterfaceSkin() {
         uiSkin.load(Gdx.files.internal("uiskin.json"));
     }
 
-    private void createModels() {
-        world = new World(WORLD_SIZE);
-        colony = new Colony(world.findEmptyPositions(PAWN_POSITIONS), new AstarPathFinder(world));
+    private void createModels() throws IOException, ClassNotFoundException {
+
+        try {
+            getModelsFromSaveFile();
+        }
+        catch (final IOException exception) {
+            world = new World(WORLD_SIZE);
+            colony = new Colony(world.findEmptyPositions(PAWN_POSITIONS),
+                                new AstarPathFinder(world));
+        }
+    }
+
+    private void getModelsFromSaveFile() throws IOException, ClassNotFoundException {
+        final LoadSystem loadSystem = new LoadSystem();
+        world = (World) loadSystem.read();
+        colony = (Colony) loadSystem.read();
```

Comment:
Maybe it would be better to have dedicated `loadWorld` and `loadColony` methods in `LoadSystem`, so that the outside client doesn't have to perform the non-typesafe cast.

Reply:
Thought process was to keep persistence as generic as possible, but guess that's already gone since we specifiy its for the save file only.

Reply:
Only "bad" thing is that they need to be loaded this way, should persistence handle it or pawntastic?

Reply:
> Yeah, that's reasonable. Oh, you could probably use a generic type parameter and cast the returned value to that type! No need for specific Colony and World methods.


Checkstyle does not like that :p Will go with loadColony and World